anta/device.py

```diff
@@ -386,6 +386,7 @@ async def _collect(self, command: AntaCommand, *, collection_id: str | None = No
         collection_id
             An identifier used to build the eAPI request ID.
         """
+        # pylint: disable=too-many-branches
```

Comment:
Yeah at some point we'll have to split up this function.

Reply:
Maybe in this PR? I think having a `_handle_eapi_command_error` method should be enough to avoid ruff/pylint complaining.

Reply:
do you want "some point" to be now? :p

Reply:
yeah will try to update here - hopefully not too many new unit tests to write!